saleor/discount/tasks.py

```diff
@@ -340,3 +340,13 @@ def set_promotion_rule_variants_task(start_id=None):
         qs = PromotionRule.objects.filter(pk__in=ids)
         fetch_variants_for_promotion_rules(rules=qs)
         set_promotion_rule_variants_task.delay(ids[-1])
+
+
+@app.task
+def set_discount_type_to_promotion_catalogue_task():
+    BATCH_SIZE = 100
```

Comment:
Did you test how long it takes to update this line discount batch? The number seems to be pretty low.